api/models/Flood.js

```diff
@@ -25,4 +25,8 @@ const FloodSchema = new Schema({
   timestamps: true,
 });
 
+FloodSchema.index({
```

Comment:
@guillaumev I am not sure a simple index by user would be enough. it might need a compound index (as in notifications). Here is the full collscan for flood:

```2019-10-29T12:26:24.533+0000 I COMMAND  [conn1023] command production.flood command: aggregate { aggregate: "flood", pipeline: [ { $match: { type: "login", email: "rosca@unfpa.org", createdAt: { $gte: new Date(1572351684305) } } }, { $group: { _id: 1, n: { $sum: 1 } } } ], cursor: {}, lsid: { id: UUID("04ac15e7-5ebc-482e-bd9e-a4d1ff7a3d9f") }, $clusterTime: { clusterTime: Timestamp(1572351980, 3), signature: { hash: BinData(0, 0000000000000000000000000000000000000000), keyId: 0 } }, $db: "production" } planSummary: COLLSCAN keysExamined:0 docsExamined:166624 cursorExhausted:1 numYields:1301 nreturned:0 reslen:230 locks:{ Global: { acquireCount: { r: 1303 } }, Database: { acquireCount: { r: 1303 } }, Collection: { acquireCount: { r: 1303 } } } protocol:op_msg 233ms```